powersimdata/network/model.py

```diff
@@ -0,0 +1,117 @@
+from importlib import import_module
+
+
+class ModelImmutables:
+    """Immutables for a grid model.
+
+    :param str model: grid model name.
+    """
+
+    def __init__(self, model):
```

Comment:
Should we add a `Constructor` docstring?

Reply:
Done